internal/pkg/core/plugin/process/stdlogger_test.go

```diff
@@ -0,0 +1,64 @@
+package process
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/elastic/elastic-agent/pkg/core/logger"
+)
+
+func Test_loggerWriter(t *testing.T) {
+	tc := []struct {
+		name string
+		args struct {
+			appName string
+			logTo   logStd
+		}
+		logMsg   string
+		logLevel zapcore.Level
+	}{
+		{name: "capture stdout",
+			args: struct {
+				appName string
+				logTo   logStd
+			}{
+				appName: "somebeats",
+				logTo:   logStdOut,
+			},
+			logMsg:   "stdout log",
+			logLevel: zapcore.InfoLevel,
+		},
+		{name: "capture stderr",
```

Comment:
nit: maybe alight the ```name``` with other properties in the structure initialization, here and above